src/QieyunEncoder/音韻地位.py

```diff
@@ -612,13 +616,138 @@ def from描述(描述: str):
 
         return 音韻地位(母, 呼, 等, 重紐, 韻, 聲)
 
-    def is_normal(self):
+    def 合法性(self) -> Tuple[str, Optional[音韻地位]]:
         '''
-        是 normal 的音韻地位。
-
-        例如，端母二等不是 normal 的音韻地位。
+        聲、韻、調組合的合法性。
+        如果有等價但更優的音韻地位能作爲替代，那麼也返回更優音韻地位，合法性取原地位和更優地位中的較差者。
+
+        合法性分爲 6 類，分別是：
+        - 無效：聲母和韻母的固有性質互相衝突，無法拼合的情況。如：徹母齊韻、昌母山韻
```

Comment:
```suggestion

        - 無效：聲母和韻母的固有性質互相衝突，無法拼合的情況。如：徹母齊韻、昌母山韻
```

列表前空一行（否則文檔生成工具 pdoc3 不認識這是列表）